client/blocks/taxonomy-manager/list-item.jsx

```diff
@@ -48,13 +50,23 @@ class TaxonomyManagerListItem extends Component {
 		this.props.onClick();
 	};
 
-	deleteItem= () => {
+	deleteItem = () => {
 		this.setState( {
 			popoverMenuOpen: false
 		} );
 		this.props.onDelete();
 	};
 
+	getTaxonomyLink() {
+		const { taxonomy, siteUrl, slug } = this.props;
+		let taxonomyBase = taxonomy;
+
+		if ( taxonomy === 'post_tag' ) {
+			taxonomyBase = 'tag';
+		}
```

Comment:
Yeah I'm not thrilled about logic like this, but we just have to work with past decisions in code sometimes 😆 

Perhaps if we find ourselves using this logic elsewhere we can abstract it out then.